examples/beam_in_vacuum/inputs_RR

```diff
@@ -0,0 +1,43 @@
+amr.n_cell = 16 16 4
+
+amr.max_level = 0
+
+my_constants.ne = 5e24
+my_constants.wp = sqrt( ne * q_e^2 / (epsilon0 * m_e))
+my_constants.E0 = wp * m_e * clight / q_e
+my_constants.kp = wp / clight
+my_constants.kp_inv = clight / wp
+
+my_constants.K = kp/sqrt(2.)
+my_constants.gamma0 = 2000
+my_constants.emittance_x = 313e-6
+my_constants.sigma_x = sqrt(emittance_x*kp_inv / sqrt(gamma0/2.) )
+my_constants.sigma_ux = emittance_x / sigma_x
+my_constants.uz = sqrt(gamma0^2 - 1 - sigma_ux^2)
+my_constants.w_beta = K*clight/sqrt(gamma0)
+
+hipace.external_E_slope = 1/2*E0/kp_inv 1/2*E0/kp_inv 0.
```

Comment:
Since we define both kp and kp_inv, let's use both.
```suggestion
my_constants.kp = wp / clight
my_constants.kp_inv = 1 / kp

my_constants.K = kp/sqrt(2.)
my_constants.gamma0 = 2000
my_constants.emittance_x = 313e-6
my_constants.sigma_x = sqrt(emittance_x*kp_inv / sqrt(gamma0/2.) )
my_constants.sigma_ux = emittance_x / sigma_x
my_constants.uz = sqrt(gamma0^2 - 1 - sigma_ux^2)
my_constants.w_beta = K*clight/sqrt(gamma0)

hipace.external_E_slope = 1/2*kp*E0 1/2*kp*E0 0.
```